dtfs-central-api/src/v1/routes/bank-routes.js

```diff
@@ -17,6 +17,24 @@ const handleExpressValidatorResult = require('../validation/route-validators/exp
 /**
  * @openapi
  * /bank:
+ *   get:
+ *     summary: Get an array of all banks in the banks collection
```

Comment:
Not for this PR - curious why the route is `/bank`, without an "S", since it is returning multiple banks.

Reply:
I thought the same thing - the route I was expecting to see was `/banks` and `/banks/:id`, but for some reason the `/bank` route was chosen instead